include/lib/cpus/aarch64/cortex_a53.h

```diff
@@ -41,6 +41,15 @@
 
 #define CPUECTLR_SMP_BIT		(1 << 6)
 
+#define CPU_RETENTION_ENTRY_2_TICKS	0x1
+#define CPU_RETENTION_ENTRY_8_TICKS	0x2
+#define CPU_RETENTION_ENTRY_32_TICKS	0x3
+#define CPU_RETENTION_ENTRY_64_TICKS	0x4
+#define CPU_RETENTION_ENTRY_128_TICKS	0x5
+#define CPU_RETENTION_ENTRY_256_TICKS	0x6
+#define CPU_RETENTION_ENTRY_512_TICKS	0x7
```

Comment:
Is it ok to just define the RETENTION_ENTRY_TICKS_xxx macros and use them directly in the code? We can have the CPUECTLR and L2ECTLR masks though.